internal/cli/output_opts.go

```diff
@@ -106,6 +106,16 @@ func (opts *OutputOpts) Print(o interface{}) error {
 	return err
 }
 
+// Print first non nil object from objectsToPrint array.
+func (opts *OutputOpts) PrintFirst(objectsToPrint ...interface{}) error {
```

Comment:
A new method that will enable us to work with multiple objects returned by store API.